contracts/DepositManager.sol

```diff
@@ -85,17 +91,14 @@ contract DepositCore is SubtreeQueue {
     }
 }
 
-contract DepositManager is DepositCore, Ownable {
+contract DepositManager is DepositCore, IDepositManager, Ownable {
     using Types for Types.UserState;
     Registry public nameRegistry;
     address public vault;
 
     Logger public logger;
     ITokenRegistry public tokenRegistry;
 
-    // batchID => subtreeRoot
-    mapping(uint256 => bytes32) public submittedSubtree;
```

Comment:
It makes more sense to let the Rollup contract keep track of which batch has deposit subtree root submitted. So that the deposit contract can be batch agnostic.